assets/src/dashboard/templates/raw/wellbeing.json

```
@@ -1349,7 +1309,11 @@
           },
           "type": "shape",
           "id": "3016e5bf-a0d2-4d9d-82f0-cb169b5e9189",
-          "link": null
+          "link": {
+            "url": "http://example.com/",
```

Nit: let's use `https://` everywhere

```suggestion
            "url": "https://example.com/",
```

---

👍